backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestOrderRepository.java

```diff
@@ -26,7 +26,8 @@
   String RESULT_RECENT_ORDER = " order by q.updatedAt desc ";
 
   @Query(FACILITY_QUERY + IS_PENDING + ORDER_CREATION_ORDER)
-  @EntityGraph(attributePaths = {"patient"})
+  @EntityGraph(
+      attributePaths = {"patient", "deviceType", "specimenType", "results", "deviceSpecimen"})
```

Comment:
In stead of `Lazy` loading this fields, this will switch it to `Eager` loading for these fields, thus removing the need for dataloaders.